text/0043-disable-sbom.md

```diff
@@ -0,0 +1,50 @@
+# Provide Global Mechanism to Disable SBOM Generation
+
+## Summary
+
+Users should have a mechanism to disable the generation of SBOM documents
+during the build process.
+
+## Motivation
+
+The generation of the SBOM documents for an image can be very time-intensive in
+some cases. We have seen a doubling of build times for Node.js applications.
+Additionally, some users are not interested in taking advantage of the SBOM
+features of the buildpacks.
+
+## Detailed Explanation
+
+If a user wishes to produce an image that does not contain SBOM documents, they
+can set the `BP_GENERATE_SBOM` environment variable to `false`. When unset, the
+value of this environment variable is assumed to be `true`.
+
+When `BP_GENERATE_SBOM` is set to `false`, none of the buildpacks executing
+during the build phase should generate or attach an SBOM in their outputs. This
+would apply to both new (Syft, CycloneDX, and SPDX formats) and old (label) SBOM
+outputs.
```

Comment:
How do you feel about specifically adding a label when the user opts out that would indicate the user opted out? I think this could be helpful for a couple of reasons:

a.) it would be easy for tooling to check this and react accordingly. The tooling would know why an SBOM isn't there.
b.) similarly, if someone is troubleshooting and SBOM isn't present, then it's easy to differentiate between it being missing due to a technical problem or the user opting out.

Reply:
Addressed in https://github.com/paketo-buildpacks/rfcs/pull/161/commits/259df7f0550f32004d063b538b4ae9cc9e51652d